### PR TITLE
Fix load and tutorial 7

### LINKED ISSUE #190: Tutorial #7 throwing ValueError: Pattern not supported: expected Constant followed by Reshape node.
```
# We'll now set up the data for our small example below
# For illustration purposes, we will create toy data
# and encrypt all of it from source ALICE
x_small = torch.rand(100, 1, 28, 28)
y_small = torch.randint(1, (100,))

# Transform labels into one-hot encoding
label_eye = torch.eye(2)
y_one_hot = label_eye[y_small]

# Transform all data to CrypTensors
x_train = crypten.cryptensor(x_small, src=ALICE)
y_train = crypten.cryptensor(y_one_hot)

# Instantiate and encrypt a CrypTen model
model_plaintext = ExampleNet()
dummy_input = torch.empty(1, 1, 28, 28)
model = crypten.nn.from_pytorch(model_plaintext, dummy_input)
model.encrypt()

---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
<ipython-input-6-c6c40d1b9ef3> in <module>
     16 model_plaintext = ExampleNet()
     17 dummy_input = torch.empty(1, 1, 28, 28)
---> 18 model = crypten.nn.from_pytorch(model_plaintext, dummy_input)
     19 model.encrypt()

/anaconda2/envs/py3/lib/python3.7/site-packages/crypten/nn/onnx_converter.py in from_pytorch(pytorch_model, dummy_input)
     35     # construct CrypTen model:
     36     f = _from_pytorch_to_bytes(pytorch_model, dummy_input)
---> 37     crypten_model = from_onnx(f)
     38     f.close()
     39 

/anaconda2/envs/py3/lib/python3.7/site-packages/crypten/nn/onnx_converter.py in from_onnx(onnx_string_or_file)
    122     """Converts an onnx model to a CrypTen model"""
    123     converter = FromOnnx(onnx_string_or_file)
--> 124     crypten_model = converter.to_crypten()
    125     return crypten_model
    126 

/anaconda2/envs/py3/lib/python3.7/site-packages/crypten/nn/onnx_converter.py in to_crypten(self)
    171             crypten_model.add_module(node_output_name, crypten_module, node_input_names)
    172 
--> 173         crypten_model = self.modify_shapes_in_graph(crypten_model)
    174         crypten_model = FromOnnx._get_model_or_module(crypten_model)
    175         return crypten_model

/anaconda2/envs/py3/lib/python3.7/site-packages/crypten/nn/onnx_converter.py in modify_shapes_in_graph(crypten_model)
    304                     raise ValueError(
    305                         """Pattern not supported: expected
--> 306                         Constant followed by Reshape node."""
    307                     )
    308                 value = crypten_modules[i - 1].value.long()

ValueError: Pattern not supported: expected
                        Constant followed by Reshape node.
```

### PR DESCRIPTION
**Pull request by gmuraru**
## Types of changes

Also, it seems there is an issue if you try to use ```torch.load``` and pass in ```kwargs```.
Received:
```
TypeError: 'src' is an invalid keyword argument for Unpickler()
```

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Docs change / refactoring / dependency upgrade

## Motivation and Context / Related issue
Fixes #190 

## How Has This Been Tested (if it applies)

- Run the tutorial

## Checklist

- [x] The documentation is up-to-date with the changes I made.
- [x] I have read the **CONTRIBUTING** document and completed the CLA (see **CONTRIBUTING**).
- [x] All tests passed, and additional code has been covered with new tests.

### CODE REVIEW

File: crypten/__init__.py [hunk 1]
```diff
@@ -330,7 +330,10 @@ def load(f, load_closure=torch.load, **kwargs):
         to the closure.
     """
     # TODO: Add support for loading from correct device (kwarg: map_location=device)
-    obj = load_closure(f, **kwargs)
+    if load_closure == torch.load:
```
Comment (gmuraru) [inline crypten/__init__.py:333]: Q: I am not sure this is the best approach to fix this.

File: tutorials/Tutorial_7_Training_an_Encrypted_Neural_Network.ipynb [hunk 2]
```diff
@@ -68,7 +68,7 @@
     "        out = self.conv1(x)\n",
     "        out = F.relu(out)\n",
     "        out = F.max_pool2d(out, 2)\n",
-    "        out = out.view(out.size(0), -1)\n",
+    "        out = out.view(-1, 16 * 12 * 12)\n",
```
Comment (gmuraru) [inline tutorials/Tutorial_7_Training_an_Encrypted_Neural_Network.ipynb:71]: https://github.com/facebookresearch/CrypTen/pull/121

File: crypten/__init__.py (hunk 1 above)
Comment (lvdmaaten) [reply crypten/__init__.py:333]: It's not pretty but it seems fine to me.

**Review by lvdmaaten — APPROVED** [review]
Thanks!

**Pull request closed without merging**